extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchMergeBufferAggregator.java

```diff
@@ -86,7 +84,7 @@ public void aggregate(final ByteBuffer buf, final int position)
       return;
     }
     final WritableMemory mem = WritableMemory.wrap(buf, ByteOrder.LITTLE_ENDIAN).writableRegion(position, size);
-    final Lock lock = stripedLock.getAt(HllSketchBuildBufferAggregator.lockIndex(position)).writeLock();
+    final Lock lock = stripedLock.getAt(StripedLockHelper.lockIndex(position)).writeLock();
     lock.lock();
     try {
       final Union union = Union.writableWrap(mem);
```

Comment:
If this statement can update the memory, please add a comment like `// Union.writableWrap(mem) can update the memory, therefore it must be inside the critical section.`. Otherwise, please move the statement outside of the critical section and add a comment like `// Union.writableWrap(mem) cannot update the memory, therefore it can be outside the critical section.`